proto/ethermint/evm/v1/query.proto

```diff
@@ -76,6 +76,14 @@ service Query {
   rpc BaseFee(QueryBaseFeeRequest) returns (QueryBaseFeeResponse) {
     option (google.api.http).get = "/evmos/evm/v1/base_fee";
   }
+
+  // GlobalMinGasPrice queries the MinGasPrice
+  // it's similar to feemarket module's method,
+  // but makes the conversion to 18 decimals
+  // when the evm denom is 6 decimals
+  rpc GlobalMinGasPrice(QueryGlobalMinGasPriceRequest) returns (QueryGlobalMinGasPriceResponse) {
```

Comment:
Doesn't this create confusion if the returned decimal is different than the one specified in the denom_decimals of the `evm.proto`? Why not returning the representation with the decimal specified with that param?